Dockerfile.agnos

```diff
@@ -176,8 +175,7 @@ COPY ./userspace/firmware/* /lib/firmware/
 # populate /etc
 COPY ./userspace/files/fstab /etc
 COPY ./userspace/files/profile /etc/profile
-COPY ./userspace/files/allow-network-control.pkla /etc/polkit-1/localauthority/50-local.d/allow-network-control.pkla
-COPY ./userspace/files/allow-modem-control.pkla /etc/polkit-1/localauthority/50-local.d/allow-modem-control.pkla
+COPY ./userspace/files/comma-polkit.rules /etc/polkit-1/rules.d/
```

Comment:
Nothing is in `/etc/polkit-1/rules.d/`, all existing rules are in `/usr/share/polkit-1/rules.d`, so using that instead